 [5/5] RUN pip install -r requirements.txt:
#9 6.834 Collecting Flask==1.1.2
#9 7.496   Downloading Flask-1.1.2-py2.py3-none-any.whl (94 kB)
#9 8.746 Collecting pythonnet==2.4.0
#9 8.989   Downloading pythonnet-2.4.0.tar.gz (1.8 MB)
#9 29.66 Collecting requests==2.22.0
#9 29.80   Downloading requests-2.22.0-py2.py3-none-any.whl (57 kB)
#9 33.05 Collecting Werkzeug>=0.15
#9 33.17   Downloading Werkzeug-2.0.1-py3-none-any.whl (288 kB)
#9 35.88 Collecting Jinja2>=2.10.1
#9 35.97   Downloading Jinja2-3.0.1-py3-none-any.whl (133 kB)
#9 37.59 Collecting click>=5.1
#9 38.24   Downloading click-8.0.1-py3-none-any.whl (97 kB)
#9 41.23 Collecting itsdangerous>=0.24
#9 41.32   Downloading itsdangerous-2.0.1-py3-none-any.whl (18 kB)
#9 43.19 Collecting chardet<3.1.0,>=3.0.2
#9 43.32   Downloading chardet-3.0.4-py2.py3-none-any.whl (133 kB)
#9 45.36 Collecting certifi>=2017.4.17
#9 45.55   Downloading certifi-2021.5.30-py2.py3-none-any.whl (145 kB)
#9 48.41 Collecting urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1
#9 48.54   Downloading urllib3-1.25.11-py2.py3-none-any.whl (127 kB)
#9 50.85 Collecting idna<2.9,>=2.5
#9 51.07   Downloading idna-2.8-py2.py3-none-any.whl (58 kB)
#9 52.84 Collecting importlib-metadata
#9 52.97   Downloading importlib_metadata-4.6.3-py3-none-any.whl (17 kB)
#9 54.42 Collecting MarkupSafe>=2.0
#9 54.66   Downloading MarkupSafe-2.0.1-cp36-cp36m-manylinux2010_x86_64.whl (30 kB)
#9 56.46 Collecting dataclasses
#9 56.71   Downloading dataclasses-0.8-py3-none-any.whl (19 kB)
#9 58.58 Collecting typing-extensions>=3.6.4
#9 58.69   Downloading typing_extensions-3.10.0.0-py3-none-any.whl (26 kB)
#9 59.83 Collecting zipp>=0.5
#9 59.99   Downloading zipp-3.5.0-py3-none-any.whl (5.7 kB)
#9 60.66 Building wheels for collected packages: pythonnet
#9 60.66   Building wheel for pythonnet (setup.py): started
#9 61.39   Building wheel for pythonnet (setup.py): finished with status 'error'
#9 61.39   ERROR: Command errored out with exit status 1:
#9 61.39    command: /usr/local/bin/python -u -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-an49r1tl/pythonnet_d97a1da067394c87a6bb74a8f6230346/setup.py'"'"'; __file__='"'"'/tmp/pip-install-an49r1tl/pythonnet_d97a1da067394c87a6bb74a8f6230346/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-k0emaxif
#9 61.39        cwd: /tmp/pip-install-an49r1tl/pythonnet_d97a1da067394c87a6bb74a8f6230346/
#9 61.39   Complete output (43 lines):
#9 61.39   running bdist_wheel
#9 61.39   running build
#9 61.39   running build_ext
#9 61.39   /bin/sh: 1: mono: not found
#9 61.39   Traceback (most recent call last):
#9 61.39     File "<string>", line 1, in <module>
#9 61.39     File "/tmp/pip-install-an49r1tl/pythonnet_d97a1da067394c87a6bb74a8f6230346/setup.py", line 658, in <module>
#9 61.39       zip_safe=False,
#9 61.39     File "/usr/local/lib/python3.6/site-packages/setuptools/__init__.py", line 153, in setup
#9 61.39       return distutils.core.setup(**attrs)
#9 61.39     File "/usr/local/lib/python3.6/distutils/core.py", line 148, in setup
#9 61.39       dist.run_commands()
#9 61.39     File "/usr/local/lib/python3.6/distutils/dist.py", line 955, in run_commands
#9 61.39       self.run_command(cmd)
#9 61.39     File "/usr/local/lib/python3.6/distutils/dist.py", line 974, in run_command
#9 61.39       cmd_obj.run()
#9 61.39     File "/tmp/pip-install-an49r1tl/pythonnet_d97a1da067394c87a6bb74a8f6230346/setup.py", line 610, in run
#9 61.39       return bdist_wheel.bdist_wheel.run(self)
#9 61.39     File "/usr/local/lib/python3.6/site-packages/wheel/bdist_wheel.py", line 299, in run
#9 61.39       self.run_command('build')
#9 61.39     File "/usr/local/lib/python3.6/distutils/cmd.py", line 313, in run_command
#9 61.39       self.distribution.run_command(command)
#9 61.39     File "/usr/local/lib/python3.6/distutils/dist.py", line 974, in run_command
#9 61.39       cmd_obj.run()
#9 61.39     File "/usr/local/lib/python3.6/distutils/command/build.py", line 135, in run
#9 61.39       self.run_command(cmd_name)
#9 61.39     File "/usr/local/lib/python3.6/distutils/cmd.py", line 313, in run_command
#9 61.39       self.distribution.run_command(command)
#9 61.39     File "/usr/local/lib/python3.6/distutils/dist.py", line 974, in run_command
#9 61.39       cmd_obj.run()
#9 61.39     File "/usr/local/lib/python3.6/distutils/command/build_ext.py", line 339, in run
#9 61.39       self.build_extensions()
#9 61.39     File "/usr/local/lib/python3.6/distutils/command/build_ext.py", line 448, in build_extensions
#9 61.39       self._build_extensions_serial()
#9 61.39     File "/usr/local/lib/python3.6/distutils/command/build_ext.py", line 473, in _build_extensions_serial
#9 61.39       self.build_extension(ext)
#9 61.39     File "/tmp/pip-install-an49r1tl/pythonnet_d97a1da067394c87a6bb74a8f6230346/setup.py", line 245, in build_extension
#9 61.39       self._install_packages()
#9 61.39     File "/tmp/pip-install-an49r1tl/pythonnet_d97a1da067394c87a6bb74a8f6230346/setup.py", line 424, in _install_packages
#9 61.39       subprocess.check_call(cmd, shell=use_shell)
#9 61.39     File "/usr/local/lib/python3.6/subprocess.py", line 311, in check_call
#9 61.39       raise CalledProcessError(retcode, cmd)
#9 61.39   subprocess.CalledProcessError: Command 'mono tools/nuget/nuget.exe update -self' returned non-zero exit status 127.
#9 61.39   ----------------------------------------
#9 61.39   ERROR: Failed building wheel for pythonnet
#9 61.39   Running setup.py clean for pythonnet
#9 62.18 Failed to build pythonnet
#9 62.57 Installing collected packages: zipp, typing-extensions, MarkupSafe, importlib-metadata, dataclasses, Werkzeug, urllib3, Jinja2, itsdangerous, idna, click, chardet, certifi, requests, pythonnet, Flask
#9 71.16     Running setup.py install for pythonnet: started
#9 71.91     Running setup.py install for pythonnet: finished with status 'error'
#9 71.92     ERROR: Command errored out with exit status 1:
#9 71.92      command: /usr/local/bin/python -u -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-an49r1tl/pythonnet_d97a1da067394c87a6bb74a8f6230346/setup.py'"'"'; __file__='"'"'/tmp/pip-install-an49r1tl/pythonnet_d97a1da067394c87a6bb74a8f6230346/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-i9rht_8j/install-record.txt --single-version-externally-managed --compile --install-headers /usr/local/include/python3.6m/pythonnet
#9 71.92          cwd: /tmp/pip-install-an49r1tl/pythonnet_d97a1da067394c87a6bb74a8f6230346/
#9 71.92     Complete output (6 lines):
#9 71.92     usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
#9 71.92        or: setup.py --help [cmd1 cmd2 ...]
#9 71.92        or: setup.py --help-commands
#9 71.92        or: setup.py cmd --help
#9 71.92
#9 71.92     error: option --single-version-externally-managed not recognized
#9 71.92     ----------------------------------------
#9 71.92 ERROR: Command errored out with exit status 1: /usr/local/bin/python -u -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-an49r1tl/pythonnet_d97a1da067394c87a6bb74a8f6230346/setup.py'"'"'; __file__='"'"'/tmp/pip-install-an49r1tl/pythonnet_d97a1da067394c87a6bb74a8f6230346/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-i9rht_8j/install-record.txt --single-version-externally-managed --compile --install-headers /usr/local/include/python3.6m/pythonnet Check the logs for full command output.
------
executor failed running [/bin/sh -c pip install -r requirements.txt]: exit code: 1
PS E:\flaskprojects\dockertest>